lib/Interpreter/CppInterOp.cpp

```diff
@@ -1212,12 +1212,20 @@ namespace Cpp {
     return 0;
   }
 
-  TCppType_t GetVariableType(TCppScope_t var)
-  {
-    auto D = (Decl *) var;
+  TCppType_t GetVariableType(TCppScope_t var) {
+    auto D = (Decl*)var;
```

Comment:
You should be able to satisfy clang-tidy with this

```suggestion
    auto *D = static_cast<Decl*>(var);
```